File: lessons/compiled_lang/java/src/k00.parameters.java
// Not public - see a00.output.java for why.
class K00Parameters {
    static void celsius(double fahrenheit) {
        double temperature = (fahrenheit - 32.0) * 5 / 9;
        System.out.printf("The Celsius temperature is %.1f degrees.\n", temperature);
    }

    public static void main(String[] args) {
        double temperature = 73;
        celsius(temperature);
    }
}

File: lessons/compiled_lang/rust/src/k00.parameters.rs
fn celsius(fahrenheit: f64) {
    let temperature = (fahrenheit - 32.0) * 5.0 / 9.0;
    println!("The Celsius temperature is {temperature:.1} degrees.");
}

fn main() {
    let temperature = 73.0;
    celsius(temperature);
}
